.editorconfig

```diff
@@ -57,3 +60,20 @@ csharp_style_allow_blank_line_after_colon_in_constructor_initializer_experimenta
 # IDE0073: The file header is missing or not located at the top of the file
 file_header_template = Licensed to the .NET Foundation under one or more agreements.\nThe .NET Foundation licenses this file to you under the MIT license.
 dotnet_diagnostic.IDE0073.severity = error
+
+# CA1305: Specify IFormatProvider
+dotnet_diagnostic.CA1305.severity = none
+
+###############################################
+# Analyzer configuration for all test C# files.
+###############################################
+[src/*/tests/**/*.cs]
```

Comment:
I opted for this option to keep the repo simple. I doubt this section will grow much. I tested the globbing pattern and it works (it does not affect the src files).

I am aware of the options we have according to the documentation: https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/configuration-files

And I now understand how we use globalconfig files in runtime: https://github.com/dotnet/runtime/pull/57144/files